### DISCUSSION TITLE
Is there any way to set TLS usage by mail server / credential ?

### DISCUSSION THREAD

**Discussion by ajtonyb** [postalserver/postal#2606] [Help with using Postal]
AFAIK there's only one way, globally, to control if TLS is used. Is there any way of setting this separately?

**Comment by willpower232** [comment]
What are you looking to accomplish? Postal mostly allows the SMTP client to decide how much TLS it wants to use.

**Comment by ajtonyb** [comment]
My question is aimed at controlling from the UI, by mail server or tenant what kind of encryption (STARTTLS or nothing) they can use.
I don't have a lot of hope for this, as choosing it is "server level", before even authenticating, but I figured it's worth a shot.

**Reply by willpower232** [reply]
Unfortunately Postal doesn't have any options for restricting what methods a credential or server can use. It is just happy to accept the message to send!

There is nothing saying you couldn't modify https://github.com/postalserver/postal/blob/main/lib/postal/smtp_server/client.rb to force it based on whatever criteria you also set up.
